src/main/java/com/ksm/wstbackoffice/entity/Address.java

```diff
@@ -0,0 +1,17 @@
+package com.ksm.wstbackoffice.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@EqualsAndHashCode
+public class Address {
+    private Country country;
```

Comment:
Let´s make address to have these properties:
```
    private long id;
    private Country country;
    private String zipcode;
    private String region;
    private String district;
    private String city;
    private String street;
    private String houseNumber;
    private String apartmentNumber;
    private String description;
```

Reply:
done